16-power-outage-monitor/README.md

```diff
@@ -114,6 +121,60 @@ This event indicates that the Notecard is no longer powered from the power brick
 
 This event indicates that the Notecard is again powered by the power brick.
 
+## Routing Data out of Notehub
+
+Notehub supports forwarding data to a wide range of API endpoints by using the Route feature. This can be used to forward your power monitoring events to external dashboards and alerts to a realtime notification service.  Here, we will use Twilio SMS API to send a notification of an alert to a phone number.
+
+For an introduction to Twilio SMS routes, please see our [Twilio SMS Guide](https://dev.blues.io/guides-and-tutorials/twilio-sms-guide/).
+
+1. Fill out the required fields for the Twilio SMS route, including "from" and "to" phone numbers, where "from" is your virtual Twilio number, and "to" is the number of the phone that receives the power outage alerts. We will not be using placeholders for these numbers, but will use a placeholder for the message, so set the message field to `[.body.customMessage]`.
+
+2. Under the "Filters" section, set "Notefiles" to "Selected Notefiles" and type `_health.qo` in the text input area below the list of notefiles. 
+
+3. Under "Data", select "JSONata Expression" and copy and paste the contents of [jsonata/route.jsonata](jsonata/route.jsonata) into the text field "Insert your JSONata expression here".
+
+4. Click "Save Changes".
+
+### Testing the Route
+
+The ideal test is to use the app Notecard to generate alerts.
+
+#### Testing Power Failure
+
+With the Notecard connected to USB power, and the LiPo connected, simulate a power outage by unplugging the USB power supply. Within a few seconds, an SMS will be sent to the `to` number you configured in the route above.
+
+The message looks like this:
+
+```
+ALERT! Power has failed to device Refrigerator.
+```
+
+This indicates a mains power failure on the Notecard with serial number "Refrigerator".
+
+#### testing Power Restored
```

Comment:
testing -> Testing

Reply:
strange, I don't see that in my local copy.